### PR TITLE
deps: bump dependencies

### PR DESCRIPTION
**Pull request by jssotomdz**
special attention on gtkmm which was limited to v4 but now some apps (system monitor for now) require the new version

### CODE REVIEW

**Review by sergio-costas — APPROVED** [review]
Looks good, although I would also check if atkmm can be updated too, because the limit version there was due to gtkmm.

**Comment by jssotomdz** [comment]
> I would also check if atkmm can be updated too

[looks](https://gitlab.gnome.org/GNOME/atkmm/-/tags) up to date

**Comment by sergio-costas** [comment]
Mmm... odd... then, why is there that "2.53.3" version limit...?

**Comment by seb128** [comment]
> Mmm... odd... then, why is there that "2.53.3" version limit...?

Seems like to workaround a bug in the upstream repository? Current version is 2.36 but

[c6ecc83d](https://gitlab.gnome.org/GNOME/atkmm/-/commit/c6ecc83d3b3b3087df465813064c06ecf3d0b92e) · [2.53.3](https://gitlab.gnome.org/GNOME/atkmm/-/commit/c6ecc83d3b3b3087df465813064c06ecf3d0b92e) · 7 years ago

File: snapcraft.yaml [hunk 1]
```diff
@@ -294,14 +294,14 @@ parts:
...
-    source-tag: '1.52.2'
+    source-tag: '1.54.0'
     source-depth: 1
     plugin: meson
     meson-parameters:
       - --prefix=/usr
       - -Doptimization=3
       - -Ddebug=true
-      - -Dinstall-tests=false
+      - -Dbuild-testsuite=false
```
Comment (seb128) [inline snapcraft.yaml:304]: The change was https://gitlab.gnome.org/GNOME/pango/-/commit/927abcc which also added -Dbuild-examples, could you set that one to false?

**Review by seb128 — CHANGES_REQUESTED** [review]
Thanks. I've one small tweak ask but it's looking good otherwise

File: snapcraft.yaml (hunk 1 above)
Comment (jssotomdz) [reply snapcraft.yaml:304]: addressed!

**Review by seb128 — APPROVED** [review]
Thanks!

**Pull request merged**